packages/odyssey-icons/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "@okta/odyssey-icons",
+  "version": "0.3.0",
+  "description": "The home for Odyssey's supported icons.",
+  "homepage": "https://github.com/okta/odyssey",
+  "license": "Apache-2.0",
+  "author": "Okta, Inc.",
+  "files": [
+    "dist"
```

Since we want to include only the `src` files:

```suggestion
    "src"
```